docs/scos/user/back-office-user-guides/202108.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -21,40 +21,37 @@ A cart rule is a discount that is applied automatically when all attached discou
-2. On the *Create Discount* page, in the *General information* tab, do the following:
-    1. In **Store relation**, check the stores you wish the discount to be active in.
-    2. From the **Discount Type** drop-down list, select a cart rule.
-    3. In the **Name** field, specify the name for the discount.
-    4. _Optional_: in the **Description** field, enter the description of the discount.
-    5. Specify if the discount is exclusive. For reference information, in the Creating a voucher guide, see the [Discount Overview page](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-overview-page) section.
-    6. Specify the validity interval (lifetime) of the discount.
-* Click **Next** or select the *Discount calculation* tab to proceed.
-
-* On the *Create Discount* page, in the *Discount calculation* tab, do the following:
-    1.  Select either Calculator percentage or Calculator fixed in the **Calculator type** drop-down. For reference information, in the Creating a voucher guide, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
+1. On the **Discount** page, in the top right corner, click **Create new discount**.
+2. On the **Create Discount** page, on the **General information** tab, do the following:
+   1. In **Store relation**, check the stores you wish the discount to be active in.
+   2. From the **Discount Type** drop-down list, select a cart rule.
+   3. In the **Name** field, specify the name for the discount.
+   4. Optional: in the **Description** field, enter the description of the discount.
+   5. Specify if the discount is exclusive. For reference information, in the **Creating vouchers** guide, see the [Discount Overview page](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-overview-page) section.
+   6. Specify the validity interval (lifetime) of the discount.
+   7. To proceed to the **Discount calculation** tab, click **Next**.
+3. On the **Create Discount** page, in the **Discount calculation** tab, do the following:
+   1. Select either Calculator percentage or Calculator fixed in the **Calculator type** drop-down. For reference information, in the **Creating vouchers** guide, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
 
         {% info_block warningBox "Note" %}
 
         The next step varies based on the selected calculator type:
-
-          a. **Calculator fixed**: Enter the prices to be discounted.
-          
-          b.  **Calculator percentage**: Enter the values (percentage) to be discounted.
+          * **Calculator fixed**: Enter the prices to be discounted.
```

Comment:
```suggestion
          * **Calculator fixed**: Enter the amount to be discounted.
```

Reply:
Prices is not the right word here. Price refers to the amount that a customer pays for an item. A discount value is not a price. Amount makes sense here because it represents a quantity, in this case a quantity of currency. Amount would never refer to a percentage value, so it distinguishes the instruction from the next option.